src/common/components/layouts/PdfDownloadOverlay.vue

```diff
@@ -190,29 +212,59 @@ export default defineComponent<Props>({
             };
         };
 
-        const createPdfWithContents = (contents: Content[]) => pdfMake.createPdf({
-            pageSize: props.paperSize,
-            pageOrientation: props.orientation,
-            pageMargins: [PAGE_PAD_X, PAGE_PAD_Y],
-            content: contents,
-            background: () => ({
-                canvas: [
-                    {
-                        type: 'rect',
-                        x: 0,
-                        y: 0,
-                        w: state.paperSizeInfo.width,
-                        h: state.paperSizeInfo.height,
-                        color: gray[100],
-                    },
-                ],
-            }),
-            styles: {
-                imageRowWrapper: {
-                    margin: [0, IMAGE_ROW_MARGIN_Y],
+        const makeTableLayouts = () => ({
+            costAnalysis: {
```

Comment:
```suggestion
            DataTable: {
```
pdf download overlay 컴포넌트는 콘솔의 특정 서비스에서 사용되는 것이 아니므로, 범용적인 네이밍이 좋을 것 같습니다!